src/argo/workflow.yaml

```diff
@@ -10,6 +10,9 @@ spec:
     - name: gcs-writer-key
       secret:
         secretName: "{{workflow.parameters.gcs-key-secret}}"
+        items:
+          - key: "{{workflow.parameters.gcs-key-name}}"
+            path: "{{workflow.parameters.gcs-key-name}}"
```

Comment:
Should the key and path be the same parameter?

Reply:
They don't necessarily need to be. Key is input (where is the thing to read from the secret?), path is output (where should the thing be written when mounted?). So we can make the path whatever we feel like, as long as it's consistent at the use sites.